csrc/u8g2.h

```diff
@@ -344,11 +352,18 @@ struct u8g2_struct
 #ifdef U8G2_WITH_HVLINE_COUNT
   unsigned long hv_cnt;
 #endif /* U8G2_WITH_HVLINE_COUNT */   
-#ifdef __unix__
+#if U8G2_WITH_GLYPH_LOOKUP_CACHE == 1
+  const uint8_t *last_unicode_glyph;
   uint16_t last_unicode;
-  const uint8_t *last_font_data;
+#else
+  uint16_t index_unicode[U8G2_INDEX_CNT_UNICODE];
+  const uint8_t *index_unicode_glyphs[U8G2_INDEX_CNT_UNICODE];
+#endif
+#ifdef U8G2_WITH_GLYPH_DATA_CACHE
+  uint8_t glyph_cache_len;
```

Comment:
The glyph data cache is for speeding up scenarios where loading from the font data is much slower than RAM.
I am implementing another feature, which allows use of font file, instead of embeded data. The advantage is much smaller sketch (for those with CJK fonts), and runtime changable fonts.
Without the glyph data cache the file based font will incur quite some slow down, since every single byte read will need to dispatch through several indirections (smart pointer refs, virtual functions, etc.)

Reply:
ah, I understand... so this is, why you use malloc(), right?
I think it should be better to allocate a fixed memory for one glyph. It will not exceed 260 bytes.